terraform/digital_ocean_ext/module/script.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+printf %s "[Unit]
+Description=Ping monitoring service
+After=network.target
+Wants=network-online.target
+
+[Service]
+Restart=always
+Type=simple
+WorkingDirectory=/opt
+ExecStart=/opt/ping
+
+[Install]
+WantedBy=multi-user.target
+" | tee /etc/systemd/system/pings.service
+
+cd /opt
+wget https://github.com/Arriven/db1000n/releases/download/v0.5.20/db1000n-v0.5.20-linux-amd64.tar.gz --output-document=/opt/db1000n-linux-amd64.tar.gz
```

Comment:
it will deploy a hardcoded version of the app. maybe would be better to download the latest version?
same for script below

Reply:
This code works for me. Of course the code could have error, however I will maintain the code until all possible bug disappear and manual will be easy to understand.

Reply:
> it will deploy a hardcoded version of the app. maybe would be better to download the latest version? same for script below

I have fixed it [my_repo](https://github.com/dddbbbsss/terraform_db1000n/blob/main/03-main.tf), but i can't update pull request here.

Reply:
> This code works for me.

but if you push with a hardcoded version then it requires maintaining or it will be dead code after several releases

> but i can't update pull request here.

you can just push new code to the same branch

Reply:
> you can just push new code to the same branch
I can't. Even the branch has fixed code, I have seen: `Your branch is up to date with 'origin/db1000n'.`